src/compat_wrappers.h

```diff
@@ -8,6 +8,10 @@
 
 #include "b-em.h"
 
+#ifndef WIN32
+#define HAVE_STPCPY
+#endif
```

Comment:
I don't see how it can be anything *other* than an inversion -- which makes me question how this is even working at all.

As for `AC_CHECK_FUNCS` -- no, that's not how it's working -- I'm still splitting that out into `AC_CHECK_FUNC` individually and setting each test manually to comply with even older autotools which I know are on systems we would claim to support.

As for test programs and `-Werror1` -- as I've asked for already, I would want to see precisely what it is about those test programs which is causing `-Werror` to choke on them, and that relies on someone putting up their `config.log` file for me to see.  We are always in control of the tests which are generated -- and the fact that they're not failing for me (on a fairly old toolchain here) makes me think it's something else.  But the `config.log` will tell me what's going on.